magpy/lib/format_iaga02.py

```diff
@@ -12,6 +12,9 @@
 
 from magpy.stream import *
 
+#global variables
+MISSING_DATA = 99999
```

Comment:
Add another global for `NOT_REPORTED`, and use that instead of hard coded eights in this module.